### PR TITLE
QC Mark All/Mark One OK.

### PR DESCRIPTION
**Pull request by local-minimum**
screenshot from 2018-05-28 11-45-58
Made all buttons have mark all except for the one with a `1`.
Added system tests for this.

### CODE REVIEW

File: scanomatic/ui_server_data/js/qc_normHelper.js [hunk 1]
```diff
@@ -171,18 +171,15 @@ function setExperimentByCoord(row, col) {
...
-    var maxQueueSize = qIndexQueue.length - 1;
-    if (qIndexCurrent < 0)
-        qIndexCurrent = maxQueueSize;
-    if (qIndexCurrent > maxQueueSize)
-        qIndexCurrent = 0;
-    var item = qIndexQueue[qIndexCurrent];
+    qIndexCurrent += operation;
+    const maxQueueSize = qIndexQueue.length - 1;
+    if (qIndexCurrent < 0) { qIndexCurrent = maxQueueSize; }
+    if (qIndexCurrent > maxQueueSize) { qIndexCurrent = 0; }
```
[the remark this answers is not in the document]
Comment (local-minimum) [reply scanomatic/ui_server_data/js/qc_normHelper.js]: out of scope/legacy code/will not fix

File: scanomatic/ui_server_data/js/qc_normHelper.js [hunk 2]
```diff
@@ -231,10 +230,14 @@ function createMarkButton(buttonId, type) {
             width: 25,
             height: 25,
         });
+    if (oneOnly) {
+        $(buttonId).append('<div class="mark-this-phenotype">1</div>');
+    }
```
[the remark this answers is not in the document]
Comment (local-minimum) [reply scanomatic/ui_server_data/js/qc_normHelper.js]: from time import sleep

[the remark this answers is not in the document]
Comment (local-minimum) [reply scanomatic/ui_server_data/js/qc_normHelper.js]: out of scope/legacy code/will not fix

File: scanomatic/ui_server_data/js/qc_normHelper.js [hunk 3]
```diff
@@ -231,10 +230,14 @@ function createMarkButton(buttonId, type) {
             width: 25,
             height: 25,
         });
+    if (oneOnly) {
+        $(buttonId).append('<div class="mark-this-phenotype">1</div>');
```
[the remark this answers is not in the document]
Comment (local-minimum) [reply scanomatic/ui_server_data/js/qc_normHelper.js]: out of scope/legacy code/will not fix

File: scanomatic/ui_server_data/js/qc_normHelper.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (local-minimum) [reply scanomatic/ui_server_data/js/qc_normHelper.js]: also, it is used

File: scanomatic/ui_server_data/js/qc_normHelper.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (local-minimum) [reply scanomatic/ui_server_data/js/qc_normHelper.js]: out of scope/legacy code/will not fix

File: scanomatic/ui_server_data/js/qc_normHelper.js [hunk 4]
```diff
@@ -231,10 +230,14 @@ function createMarkButton(buttonId, type) {
             width: 25,
             height: 25,
         });
+    if (oneOnly) {
```
[the remark this answers is not in the document]
Comment (local-minimum) [reply scanomatic/ui_server_data/js/qc_normHelper.js]: out of scope/legacy code/will not fix/in global score

File: scanomatic/ui_server_data/js/qc_normHelper.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (local-minimum) [reply scanomatic/ui_server_data/js/qc_normHelper.js]: out of scope/legacy code/will not fix/global scope

File: scanomatic/ui_server_data/js/qc_normHelper.js [hunk 5]
```diff
@@ -231,10 +228,14 @@ function createMarkButton(buttonId, type) {
...
+        $(buttonId).append('<div class="mark-this-phenotype">1</div>');
+    }
```
[the remark this answers is not in the document]
Comment (local-minimum) [reply scanomatic/ui_server_data/js/qc_normHelper.js]: out of scope/legacy code/will not fix/global scope

File: tests/system/conftest.py [hunk 6]
```diff
@@ -120,3 +130,13 @@ def pytest_runtest_makereport(item, call):
     # set an report attribute for each phase of a call, which can
     # be "setup", "call", "teardown"
     setattr(item, "{}_result".format(result.when), result)
+
+
+@pytest.fixture(scope='function')
+def with_analysis():
```
Comment (gdetrez) [inline tests/system/conftest.py]: This name is a bit confusing: what is with analysis?

**Pull request merged**